org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/GradleBuild.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2018 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.buildship.core;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+
+/**
+ * Represents a Gradle build.
+ *
+ * @author Donat Csikos
+ * @since 3.0
+ */
+public interface GradleBuild {
+
+    /**
+     * Synchronizes the workspace with this Gradle build.
+     *
+     * <p>
+     * The method loads the Gradle build configuration and updates the workspace based on the
+     * retrieved information.The algorithm is as follows:
+     * <ul>
+     *   <li>Synchronize all Gradle projects of the Gradle build with the Eclipse workspace
+     *       project counterparts. If there is a project in the workspace at the location of the
```

Comment:
"If there is a project in the workspace at the location of theGradle project..."

And if there isn't, we import it.

Reply:
Added clarification to the section.